metecho/api/admin.py

```diff
@@ -45,6 +48,25 @@ def clean(self):
             )
 
 
+class GitHubOrganizationForm(forms.ModelForm):
+    class Meta:
+        model = GitHubOrganization
+        exclude = ()
+
+    def clean_login(self):
+        login = self.cleaned_data["login"]
+        try:
+            gh.get_org_for_repo_creation(orgname=login)
```

Comment:
@jerivas I notice that the mockup also calls for displaying an org's avatar. I assume that's info we can get automatically from GH -- would it make sense to fetch it here and store it in the DB?

Reply:
Fixed in ecc050a43b31c00ae74ec4c9c1c86e763da949bf